### ISSUE TITLE
Support more notification types

### ISSUE THREAD

**Issue by nexushoratio** [nexushoratio/userscripts#42]
After turning on every notification type, I now have a very full notification page, including some that fail to trigger.

Will just use this as a catch-all bug for all I currently have.

**Comment by nexushoratio** [comment]
I just found one notification card where the embedded link and the button do different things.  The card is `N people viewed your profile` and the link takes the user to me/profile-views.  But the button is `Try Premium Free for 1 Month` and takes the user to premium/survey.

In this particular case, I'm not to excited about supporting the button over the link.  But I suppose it does mean there could be others like that to keep an eye out for.

**Comment by nexushoratio** [comment]
One thing I had been keeping an eye out for is a notification with a `button.message-anywhere-button`.  Trying to figure out if it could be matched with a `a.nt-card__headline`.  Turns out no.  The example I've seen is `Reach out to your recent connection` with a `Message` button.

What is amusing is that, in most cards, the text is a link.  In this case, the text and the button are both button elements with the `.message-anywhere-button` class.  Talk about inconsistency!

I had to double check to see why the debugging code didn't throw an alert.  I certainly expected it to.
